### DISCUSSION TITLE
Audio stops working when connecting to another screen that is the same brand/model.

### DISCUSSION THREAD

**Discussion by sebszocinski** [MonitorControl/MonitorControl#1689] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

It appears as though when using this app with two monitors that are exactly the same (Home and Office) it stops working. No videos are able to play at all because the Audio isn't working. I can't seem to fix it either.

### Steps to reproduce

Use the app across two monitors via Thunderbolt connection

### Expected behavior

It should detect each monitor

### Environment Information (please complete the following information)

```markdown
- macOS version:
- Mac model:
- MonitorControl version:
- Monitor(s):
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
The app should not interfere with audio output at all. For volume control it uses DDC but that does not affect the audio stream and is unrelated to how the macOS audio subsystem works. Are you sure this issue is app related?
